lib/agent.js

```diff
@@ -171,6 +171,14 @@ Agent.prototype.start = function (opts) {
   return this
 }
 
+Agent.prototype.setFramework = function ({ name, version, overwrite = true }) {
```

Comment:
Any particular reason you chose to use an object here, versus just the three things as parameters? Anticipating collecting more info in the future?

Reply:
No, my main concern was just not to have a boolean as a standalone argument as that's not really that user-friendly. How about something like this?

```js
function (name, version, { overwrite = true }) {...}
```

Reply:
The main reason why I went with this API was so that you could provide `version` without having to provide the `name`

Reply:
I think it's fine as-is. Just wondered if there was a particular reason for the design which I was missing.